easybuild/tools/include.py

```diff
@@ -142,6 +143,11 @@ def verify_imports(pymods, pypkg, from_path):
         _log.debug("Import of %s from %s verified", pymod_spec, from_path)
 
 
+def is_software_specific_easyblock(module):
+    """Determine whether Python module at specified location is a software-specific easyblock."""
+    return bool(re.search('^class EB_.*\(.*\):$', read_file(module), re.M))
```

Comment:
keep accidental trailing whitespace in mind?

Reply:
Makes sense, done